src/App.tsx

```diff
@@ -1,61 +1,134 @@
 import './App.scss';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { Todos } from './types/Todos';
+import { User } from './types/User';
+
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
```

Comment:
Consider renaming the function `getUser` to `findUserById` for better readability and understanding of its purpose.